src/app/settings.py

```diff
@@ -0,0 +1,140 @@
+"""
+Django settings for app project.
+
+Generated by 'django-admin startproject' using Django 2.0.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.0/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.environ.get('django_secret')
+
+SOCIAL_AUTH_MEDIAWIKI_KEY = os.environ.get('mediawiki_key')
+SOCIAL_AUTH_MEDIAWIKI_SECRET = os.environ.get('mediawiki_secret')
+SOCIAL_AUTH_MEDIAWIKI_URL = 'https://meta.wikimedia.org/w/index.php'
+SOCIAL_AUTH_MEDIAWIKI_CALLBACK = os.environ.get('mediawiki_callback')
```

Comment:
I've set up the consumer in such a way that it doesn't default to oob and I need to explicitly pass the value. And since Django app requires this variable, I have set it as Django's environment variable. I took reference from this doc: https://wikitech.wikimedia.org/wiki/Help:Toolforge/My_first_Django_OAuth_tool#Login_and_deploy
In order to not make it an environment variable, I need to set up the consumer again, should I go for it?